### PR TITLE
feat(microsoft-word): init

### PR DESCRIPTION
**Pull request by GitMuslim**
## 🎉 Theme for Microsoft Word 🎉

I use it a lot for school stuff so why not

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

**Comment by GitMuslim** [comment]
oh btw I currently themed word only since I don't use others rly

**Comment by GitMuslim** [comment]
@isabelroses undraft it

File: styles/microsoft-word/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select flavour "Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+
+==/UserStyle== */
+
+@-moz-document domain('live.com') {
```
Comment (uncenter) [inline styles/microsoft-word/catppuccin.user.css:18]: The domain is live.com here but in userstyles.yml it is office.com. Not sure which is the right one but we should just have a single URL.

Comment (GitMuslim) [reply styles/microsoft-word/catppuccin.user.css:18]: solved in discord discussion

File: styles/microsoft-word/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/microsoft-word
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/microsoft-word/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Amicrosoft-word
+@description Soothing pastel theme for Microsoft Word
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select flavour "Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
```
Comment (uncenter) [inline styles/microsoft-word/catppuccin.user.css:13]: Dark/light mode?

Comment (GitMuslim) [reply styles/microsoft-word/catppuccin.user.css:13]: I think it wasn't possible for *all* of ms365 but I believe word does have a way, will try to implement it later.

**Comment by isabelroses** [comment]
I only found 1 issue when testing, in the most left pop out navbar, this section appears unthemed:
image

**Review by isabelroses — APPROVED** [review]

**Pull request merged**